packages/web-components/src/components/va-date/va-date.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../../src/mixins/accessibility.css';
+@import '../../mixins/form-field-error.scss';
```

Comment:
Re-ordered these imports because changing form-field-error to a scss file made it so that some styles from uswds-error-border were being overwritten. Mainly the padding was rendering as 1.25rem instead of 1rem. 

This is also the case for memorable-date and telephone-input.